### PR TITLE
[BZ1684657]: Add a KCS and a note

### PR DESCRIPTION
**Pull request by xenolinux**
Adds a note about encountering an error while running `yum install openshift-ansible` command. The note contains a link to the KCS.
[Bug](https://bugzilla.redhat.com/show_bug.cgi?id=1684657)
[preview](https://deploy-preview-42483--osdocs.netlify.app/openshift-enterprise/latest/install/host_preparation#installing-base-packages)
OCP version: 3.11
QE contact: @jianlinliu LGTMed

### CODE REVIEW

File: install/host_preparation.adoc [hunk 1]
```diff
@@ -283,6 +283,11 @@ install the following package:
 # yum install openshift-ansible
 ----
 +
+[NOTE]
+====
+If you face an error while running `yum install openshift-ansible` command, you might need an additional repository to proceed with the installation. For more information, see https://access.redhat.com/solutions/3949501.
+====
++
```
Comment (jianlinliu) [inline install/host_preparation.adoc:290]: Per the KCS link, the issue is already fixed in the latest release, I do not think using an additional repository is a good idea, but only link customers to reference this KCS link.

Comment (xenolinux) [reply install/host_preparation.adoc:290]: Addressed

File: install/host_preparation.adoc [hunk 2]
```diff
@@ -283,6 +283,11 @@ install the following package:
 # yum install openshift-ansible
 ----
 +
+[NOTE]
+====
+If you face an error while running `yum install openshift-ansible` command, see https://access.redhat.com/solutions/3949501.
```
Comment (sagidlow) [inline install/host_preparation.adoc:288]:
```suggestion
If you encounter an error while running `yum install openshift-ansible` command, see https://access.redhat.com/solutions/3949501.
```

Using the word "face" may have translation issues, I would advise using encounter.

Comment (xenolinux) [reply install/host_preparation.adoc]: Addressed

File: install/host_preparation.adoc [hunk 3]
```diff
@@ -283,6 +283,11 @@ install the following package:
 # yum install openshift-ansible
 ----
 +
+[NOTE]
+====
+If you encounter an error while running `yum install openshift-ansible` command, see https://access.redhat.com/solutions/3949501.
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply install/host_preparation.adoc]: Addressed

[the remark this answers is not in the document]
Comment (xenolinux) [reply install/host_preparation.adoc]: Addressed

File: install/host_preparation.adoc [hunk 4]
```diff
@@ -283,6 +283,11 @@ install the following package:
 # yum install openshift-ansible
 ----
 +
+[NOTE]
+====
+If you encounter an error while running `yum install openshift-ansible` command, see link:https://access.redhat.com/solutions/3949501[the resolution to fix that error].
```
Comment (bobfuru) [inline install/host_preparation.adoc:288]:
```suggestion
If you encounter an error while running the `yum install openshift-ansible` command, see link:https://access.redhat.com/solutions/3949501[the resolution to fix that error].
```

**Review by bobfuru — COMMENTED** [review]
One small nit. I am having trouble with the netlify preview (might be system-wide presently), so I'm unable to review the rendering. Assuming you ran a local build to check that it looks good, everything else LGTM.

File: install/host_preparation.adoc (hunk 4 above)
Comment (xenolinux) [reply install/host_preparation.adoc]: Addressed

**Pull request merged**